src/ciceroscm/ciceroscm.py

```diff
@@ -84,21 +86,40 @@ def __init__(self, cfg):
               or emission runs, and start and end of run etc.
 
         """
+
         self.cfg = cut_and_check_pamset(
-            {"nystart": 1750, "nyend": 2100, "emstart": 1850, "idtm": 24}, cfg
+            {"nystart": 1750, "nyend": 2100, "emstart": 1850, "idtm": 24},
+            cfg,
+            {"carbon_cycle_model": "default", "thermal_model": "default"},
         )
         cfg.update(self.cfg)
+
         input_handler = InputHandler(cfg)
         self.cfg["rf_run"] = input_handler.optional_pam("forc")
+        if input_handler.thermal_model(self.cfg):
+            self.cfg["thermal_model"] = input_handler.thermal_model(self.cfg)
+        else:
+            self.cfg["thermal_model"] = "default"
+        if input_handler.carbon_model(self.cfg):
+            self.cfg["carbon_cycle_model"] = input_handler.carbon_model(self.cfg)
+        else:
```

Comment:
Maybe just have the input_handler methods return "default" rather than "False" when the configuration is missing, so we don't need all this if-else logic here?